libethashseal/Ethash.cpp

```diff
@@ -84,10 +84,13 @@ StringHashMap Ethash::jsInfo(BlockHeader const& _bi) const
 
 EVMSchedule const& Ethash::evmSchedule(EnvInfo const& _envInfo) const
 {
-	if (_envInfo.number() >= chainParams().u256Param("frontierCompatibilityModeLimit"))
-		return HomesteadSchedule;
+	if (_envInfo.number() >= chainParams().u256Param("EIP150ForkBlock"))
+		return EIP150Schedule;
 	else
-		return FrontierSchedule;
+		if (_envInfo.number() >= chainParams().u256Param("homsteadForkBlock"))
```

Comment:
Sure, but your comment "in eth when parsing json config this field is called frontierCompatibilityModeLimit other then that its fine" had a point, people might have such config files.